src/FakeItEasy/Configuration/RuleBuilder.cs

```diff
@@ -63,7 +63,6 @@ public IVoidConfiguration WhenArgumentsMatch(Func<ArgumentCollection, bool> argu
 
         public virtual IAfterCallSpecifiedConfiguration DoesNothing()
         {
-            this.RuleBeingBuilt.Applicator = x => { };

```

Comment:
Are you talking about [these specs](https://github.com/blairconrad/FakeItEasy/blob/49ab38fd9fc06c4e5c00b2d2870e318e0178f9f6/tests/FakeItEasy.Specs/ConfigurationSpecs.cs#L495-L534)?

I added them when I fixed my fixed. I figured since @thomaslevesque had asked about them, I might as well add evidence that it worked the way I thought while I was at it.